controller.go

```diff
@@ -270,6 +285,21 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 
 	network.processOptions(options...)
 
+	if !c.started {
+		// Global stores are watched only when the controller enters the 'started' state.
```

Comment:
Does this comment assume too much about how the caller is using libnetwork? Why can't globalNetworks before Start? Even assuming existing docker side implementation what happens when the default network is configured to be a global network? Also I don't see the equivalent reconciliation for endpoints